lib/plugins/aws/utils/pickWebsocketsTemplatePart.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
It's websockets specific so let's keep it at `lib/plugins/aws/package/compile/events/websockets/lib`